test/helpers/application_helper_test.rb

```diff
@@ -0,0 +1,27 @@
+require "test_helper"
+
+class ApplicationHelperTest < ActionView::TestCase
+  
+test "only at most 2 initials with default limit" do
+    assert_equal "jz", only_user_initials("jdz")
+    assert_equal "pl", only_user_initials("Plllll")
```

Comment:
It’s a bit unexpected that this method would change case. The way I wrote it preserves whatever case was passed in. I think that’s what most users would expect.

Also, this second test is bad because the fact that it returns “pl” does not confirm that it’s returning the first letter and the last letter. There are a lot of “l”s

Reply:
No longer changing case. Took out a few l's.

Reply:
True, does not change case now.